rero_ils/modules/items/api.py

```diff
@@ -824,7 +823,7 @@ def automatic_checkin(self):
                 LoanAction.NO: None
             }
 
-        elif self.status == ItemStatus.MISSING:
+        if self.status == ItemStatus.MISSING:
```

Comment:
If you not using `elif` the `self.status`test will be rerun even the status is `IN_TRANSIT`

Reply:
fixed